### PR TITLE
Add CI configuration for ROCm

### PR DESCRIPTION
**Pull request by okuta**
This PR is for #1094 .

### CODE REVIEW

File: .pfnci/config.pbtxt [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+
+configs {
+  key: "cupy.py37.amd"
+  value {
+    requirement {
+      cpu: 2
+      memory: 10
+    }
+    time_limit: {
+      seconds: 900
```
Comment (kmaehashi) [inline .pfnci/config.pbtxt]: Build timeout happening in CI.

**Comment by okuta** [comment]
I fixed.

**Comment by kmaehashi** [comment]
Will merge after confirming https://ci.preferred.jp/cupy.py3.amd/41454/ succeeds.

**Comment by kmaehashi** [comment]
Will merge after #2632 got merged.

**Comment by kmaehashi** [comment]
Merging as I confirmed that the test succeeds with https://ci.preferred.jp/cupy.py3.amd/41936/.

**Comment by kmaehashi** [comment]
I've activated CI webhook for PRs.

**Pull request merged**